include/osquery/database.h

```diff
@@ -50,6 +50,9 @@ extern const std::string kQueries;
 /// The "domain" where event results are stored, queued for querytime retrieval.
 extern const std::string kEvents;
 
+/// The "domain" where the results of carve queries are stored.
+extern const std::string kCarves;
```

Comment:
I think the desire here was to keep carves separated out from traditional queries. I can move this back into `kQueries` if we'd like, but this was mostly just a residual issue from the first PR bringing the initial functionality. As it stands currently querying for all executed `carves` will give you nothing as the domain doesn't yet exist.